keyboards/mechwild/mercutio/keymaps/jonavin/keymap.c

```diff
@@ -147,14 +247,15 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
             }
             oled_set_cursor(8,3);
             if (get_highest_layer(layer_state) == selected_layer) {
-                oled_write_P(PSTR("            "), false);
+                oled_write_P(PSTR("             "), false);
             } else {
                 switch (get_highest_layer(layer_state)) {
                     case 0:
                         oled_write_P(PSTR("Temp BASE"), false);
                         break;
                     case 1:
-                        oled_write_P(PSTR("Temp FN"), false);
+                        sprintf(fn_str, "Temp FN %5s", selectedkey_rec.keydesc);
```

Comment:
just a heads up, snprintf will use u about 1.5kb of firmware space, when it's used.

Reply:
Do you recommend using strncat here instead?